k8s/upgrades/README.md

```diff
@@ -175,17 +177,20 @@ using Kubernetes Job spec.
 
 The following instructions provide details on how to create your Upgrade Job specs.
 Please ensure the `from` and `to` versions are as per your upgrade path. The below
-examples show upgrading from 1.0.0 to 1.8.0.
+examples show upgrading from 1.0.0 to 1.9.0.
 
 ### Upgrade the OpenEBS Jiva PV
 
-**Note: Scaling down the application will speed up the upgrade process. It is highly recommended to scale down the application if upgrading from 1.5.0 or earlier versions of the volume.**
+**Note:Scaling down the application will speed up the upgrade process and prevent any read only issues. It is highly recommended to scale down the application if upgrading from 1.8.0 or earlier versions of the volume.**
```

Comment:
what is the point of mentioning 'if upgrading from 1.8.0 or earlier versions of the volume'? we can remove that 'if'